environments/prod.yml

```diff
@@ -26,6 +26,6 @@ probes:
 
 resources:
   limits:
-    memory: 900Mi
+    memory: 1300Mi
   requests:
-    memory: 1200Mi
+    memory: 1100Mi
```

Comment:
I noticed in the PR for staging, we also bumped down the memory for requests. I'm assuming we had it set a little too high?

Reply:
I am embarrassed to say I originally had them backwards... The limit should be higher than the requests. For prod Kanopy has the "suggested" requests for docs-search-transport as 1.04 GB, and limit to be 1.14 GB. I decided to bump that a bit. [Here is the grafana](https://grafana.corp.mongodb.com/d/a164a7f0339f99e89cea5ab47f9be872/memory-profiler-workloads?orgId=1&var-datasource=default&var-namespace=docs&var-workload=docs-search-transport-web-app&var-type=Deployment&from=now-6M&to=now&refresh=15m) for prod as reference!